cognite/client/data_classes/filters.py

```diff
@@ -202,6 +202,21 @@ def _involved_filter_types(self) -> set[type[Filter]]:
                 output.update(filter_._involved_filter_types())
         return output
 
+    def __list_filters_without_nesting(self, right_flt: Filter, operator: type[CompoundFilter]) -> list[Filter]:
```

Comment:
This can just be a single underscore method:
```suggestion
    def _list_filters_without_nesting(self, right_flt: Filter, operator: type[CompoundFilter]) -> list[Filter]:
```